plugins/sip/list.json

```diff
@@ -52,7 +52,7 @@
             "mainColor": "#4D4C4F"
         },
         {
-            "name": "Bouygues Télécom",
+            "name": "Bouygues T??l??com",
```

Comment:
@fouille why ?

Reply:
add new commit in same instant for this correction :-)